tools/install-powershell.ps1

```diff
@@ -38,7 +38,7 @@ $ErrorActionPreference = "Stop"
 
 $IsLinuxEnv = (Get-Variable -Name "IsLinux" -ErrorAction Ignore) -and $IsLinux
 $IsMacOSEnv = (Get-Variable -Name "IsMacOS" -ErrorAction Ignore) -and $IsMacOS
-$IsWinEnv   = !$IsLinuxEnv -and !$IsMacOSEnv
+$IsWinEnv = !$IsLinuxEnv -and !$IsMacOSEnv
```

Comment:
You could have retained indentation.
```
$IsMacOSEnv = (Get-Variable -Name "IsMacOS" -ErrorAction Ignore) -and $IsMacOS
$IsWinEnv      = !$IsLinuxEnv -and !$IsMacOSEnv
```

Reply:
Thank you, don't know how i didnt see that sending this in.